test/src/test/testtool/Validators/Schema/SoapValidator.wsdl

```diff
@@ -1,444 +1,543 @@
-              </xsd:sequence>
-            </xsd:complexType>
-          </xsd:element>
-        </xsd:sequence>
-      </xsd:complexType>
-    </xsd:schema>
-  </wsdl:types>
-  <wsdl:message name="Message_Item_Request">
-    <wsdl:part name="Part_Item_Request" element="ns2:Item_Request"/>
-  </wsdl:message>
-  <wsdl:message name="Message_Item_Response">
-    <wsdl:part name="Part_Item_Response" element="ns2:Item_Response"/>
-  </wsdl:message>
-  <wsdl:portType name="PortType_SoapValidator">
-    <wsdl:operation name="Operation___wsdl.SoapValidator.listener_of__SoapValidator-Webservice_.soapAction_">
-      <wsdl:input message="tns:Message_Item_Request"/>
-      <wsdl:output message="tns:Message_Item_Response"/>
-    </wsdl:operation>
-  </wsdl:portType>
-  <wsdl:binding name="Binding_SoapValidator" type="tns:PortType_SoapValidator">
-    <soap:binding transport="http://schemas.xmlsoap.org/soap/http" style="document"/>
-    <wsdl:operation name="Operation___wsdl.SoapValidator.listener_of__SoapValidator-Webservice_.soapAction_">
-      <soap:operation style="document" soapAction="${wsdl.SoapValidator.listener of [SoapValidator-Webservice].soapAction}"/>
-      <wsdl:input>
-        <soap:body parts="Part_Item_Request" use="literal"/>
-      </wsdl:input>
-      <wsdl:output>
-        <soap:body parts="Part_Item_Response" use="literal"/>
-      </wsdl:output>
-    </wsdl:operation>
-  </wsdl:binding>
-  <wsdl:service name="Service_SoapValidator">
-    <wsdl:port name="Port_SoapValidator" binding="tns:Binding_SoapValidator">
-      <soap:address location="http://localhost:8080/iaf-test/services/"/>
-    </wsdl:port>
-  </wsdl:service>
-</wsdl:definitions>
\ No newline at end of file
+	<wsdl:documentation>Generated by FrankConsole as SoapValidator.wsdl on 2021-04-30 15:09:55.</wsdl:documentation>
+	<wsdl:types>
+		<!-- v0.1.2 -->
+		<xsd:schema xmlns="http://nn.nl/XSD/Generic/MessageHeader/1" attributeFormDefault="unqualified" elementFormDefault="qualified" targetNamespace="http://nn.nl/XSD/Generic/MessageHeader/1">
+	<xsd:element name="MessageHeader" type="MessageHeaderType"/>
+	<xsd:element name="Result" type="ResultType"/>
+	<xsd:complexType name="MessageHeaderType">
+		<xsd:sequence>
+			<xsd:element name="From" type="ProducingEndpointType"/>
+			<xsd:element name="To" type="ConsumingEndpointType"/>
+			<xsd:element name="HeaderFields" type="HeaderFieldType"/>
+			<xsd:element minOccurs="0" name="ConnectionInfo" type="ConnectionInfoType"/>
+			<xsd:element name="Service" type="ServiceType"/>
+			<xsd:element minOccurs="0" name="TraceHeaderList" type="TraceHeaderListType"/>
+			<xsd:element minOccurs="0" name="TransactionList" type="TransactionListType"/>
+		</xsd:sequence>
+	</xsd:complexType>
+	<xsd:complexType name="ProducingEndpointType">
+		<xsd:sequence>
+			<xsd:element name="Id" type="xsd:string">
+				<xsd:annotation>
+					<xsd:documentation>
+						Unique identification of the initiator.
+					</xsd:documentation>
```

Comment:
Could you trim() these documentations, so that there is no useless whitespace?

Reply:
I'll cheat a bit here

Reply:
I see. This is just copied from the input XSD.